package.json

```
@@ -6,7 +6,7 @@
     "Rusty Bailey <rustylbailey@gmail.com>",
     "Gustavo Henke <guhenke@gmail.com>"
   ],
-  "version": "6.4.1",
+  "version": "6.5.0",
```

I'll upgrade separately.

```suggestion
  "version": "6.4.1",
```